packages/es-components/src/components/base/skeletonTile/SkeletonTile.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const TileFlex = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 17px 24px 0 14px;
+`;
+
+const HeaderFlex = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 20px;
+`;
+
+const IconPlaceholder = styled.div`
+  height: 65px;
+  width: 65px;
+  flex-grow: 0;
+  background-color: ${props => props.theme.colors.gray3};
+  margin-right: 40px;
+`;
+
+const TitlePlaceholder = styled.div`
+  height: 26px;
+  flex-grow: 1;
+  align-self: center;
+  background-color: ${props => props.theme.colors.gray3};
+`;
+
+const DescriptionPlaceholder = styled.div`
+  height: 15px;
+  background-color: ${props => props.theme.colors.gray3};
+  margin-bottom: 9px;
+`;
+
+const ButtonPlaceholder = styled.div`
+  height: 17px;
+  width: 158px;
+  padding: 9px 10px;
+  background-color: ${props => props.theme.colors.gray3};
+  background-clip: content-box;
+  border: 1px solid ${props => props.theme.colors.gray5};
+  border-radius: 1px;
+  align-self: flex-end;
+  margin-top: 10px;
+`;
+
+const LoadingLight = styled.div`
+  &::after {
+    content: '';
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    background-size: 50%;
+    background-repeat: no-repeat;
+    background-image: linear-gradient(
+      -45deg,
+      rgba(255, 255, 255, 0) 0,
+      rgba(255, 255, 255, 0) 45%,
+      rgba(255, 255, 255, 0.8) 50%,
+      rgba(255, 255, 255, 0) 55%,
+      rgba(255, 255, 255, 0) 100%
+    );
+    animation: loading 2.5s linear infinite;
+  }
+
+  @keyframes loading {
+    from {
+      background-position: -150% 0;
+    }
+    to {
+      background-position: 250% 0;
+    }
+  }
+`;
+
+const TileContainer = styled.div`
+  overflow: auto;
+  position: relative;
+`;
+
+function SkeletonTile() {
+  return (
+    <TileContainer>
+      <LoadingLight />
+
+      <TileFlex>
+        <HeaderFlex>
+          <IconPlaceholder />
+          <TitlePlaceholder />
+        </HeaderFlex>
+        <DescriptionPlaceholder />
+        <DescriptionPlaceholder />
+        <ButtonPlaceholder />
+      </TileFlex>
+    </TileContainer>
```

Comment:
Maybe what we should export is a `LoadingSkeleton` container which can take a shimmer color, and a shape color, and a `SkeletonShape`, which will be the correct color inside the Skeleton, but can be formed any way the downstream user wants.

So our example might be a tile, but it's just a tile built up of a `LoadingSkeleton` and `SkeletonShape`s.

Reply:
We should be able to use [TinyColor](https://www.npmjs.com/package/tinycolor2), which we already depend on, to do shapes and shimmers with any kind of color passed in, and convert those to rgba where necessary.